doc/conf.py

```diff
@@ -292,7 +292,18 @@
         'sklearn': 'http://scikit-learn.org/stable'}
     }
 
+
+def touch_example_backreferences(app, what, name, obj, options, lines):
```

Comment:
I was also thinking about it. In general you don't need this. It is only to remove some warnings of sphinx as it runs autosummary.